frontend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:lts-buster-slim
```

Comment:
1.  `add #syntax = docker/dockerfile:experimental` is missing at the beginning of the files, which causes mount on line 8 to throw errors.

2. frontend throws errors. Does node:lts-buster-slim point correspond to node14 ?
```
frontend_1           | Failed to compile.
frontend_1           | 
frontend_1           | ./src/styles/index.scss (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-6-1!./node_modules/postcss-loader/src??postcss!./node_modules/resolve-url-loader??ref--5-oneOf-6-3!./node_modules/sass-loader/dist/cjs.js??ref--5-oneOf-6-4!./src/styles/index.scss)
frontend_1           | Error: Missing binding /code/node_modules/node-sass/vendor/linux-x64-83/binding.node
frontend_1           | Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 14.x
```

Reply:
I will remove the target cache stuff as they are not very useful in this context. They lead to very weird conflicts, instead we will make use of the Docker layering cache.